modules/ROOT/pages/configure-partner.adoc

```diff
@@ -74,7 +74,3 @@ View information about an uploaded certificate by clicking it's link. See xref:C
 * Character sets
 
 Refer to the options for each at xref:x12-receive-read-settings.adoc[X12 Settings - Receive from Partner]
```

Comment:
Suggest omitting, as all xrefs in tasks.

Reply:
Let's look at that next release.